src/layouts/EditHomepage/EditHomepage.jsx

```diff
@@ -723,6 +736,14 @@ const EditHomepage = ({ match }) => {
             err
           )
           setHomepageState(updatedHomepageState)
+          setAnnouncementScrollRefs(
+            update(announcementScrollRefs, {
+              $push: [createRef()],
+            })
+          )
+          // We know this is the case since announcements are added from the top
+          const newAnnouncementIndex = 0
+          scrollTo(announcementScrollRefs[newAnnouncementIndex])
```

Comment:
this might no longer be applicable here

Reply:
fixed [ea67f34](https://github.com/isomerpages/isomercms-frontend/pull/1530/commits/ea67f34ea4ab01860270cb1da1787a1e75f5cef6)